config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
-if ENV['RAILS_USE_MEMORY_STORE'] || (!Rails.env.development? && !Rails.env.production?)
+if ENV['RAILS_USE_MEMORY_STORE'] || TokenStore.use_memory?
```

Comment:
I don't understand this change.  This might cause regular UI sessions to be stored in memory in non-test environments, which doesn't seem right.